css/_variables.scss

```diff
@@ -220,6 +220,13 @@ $welcomePageButtonFontWeight: inherit;
 $welcomePageButtonBorderRadius: 4px;
 $welcomePageButtonLineHeight: 35px;
```

Comment:
@muscat1 In order to prevent your work from being disturbed with this PR, I left these `$welcomePageButton*` variables as they are. Hope this helps :-)